charts/timesketch/README.md

```diff
@@ -0,0 +1,286 @@
+| `postgresql.primary.service.ports.postgresql`      | PostgreSQL primary service port                                             | `5432`       |
+| `postgresql.primary.persistence.size`              | PostgreSQL Persistent Volume size                                           | `8Gi`        |
+| `postgresql.primary.resources.limits`              | The resources limits for the PostgreSQL primary containers                  | `{}`         |
+| `postgresql.primary.resources.requests`            | The requested resources for the PostgreSQL primary containers               | `{}`         |
+| `postgresql.readReplicas.replicaCount`             | Number of PostgreSQL read only replicas                                     | `0`          |
+| `postgresql.readReplicas.service.type`             | PostgreSQL read replicas service type                                       | `ClusterIP`  |
+| `postgresql.readReplicas.service.ports.postgresql` | PostgreSQL read replicas service port                                       | `5432`       |
+| `postgresql.readReplicas.persistence.size`         | PostgreSQL Persistent Volume size                                           | `8Gi`        |
+| `postgresql.readReplicas.resources.limits`         | The resources limits for the PostgreSQL read only containers                | `{}`         |
+| `postgresql.readReplicas.resources.requests`       | The requested resources for the PostgreSQL read only containers             | `{}`         |
+
+Specify each parameter using the --set key=value[,key=value] argument to helm 
+install. For example,
+
+```console
+helm install my-release \
+    --set opensearch.replicas=3
+    oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/timesketch
+```
+
+The above command installs Timesketch with 3 Opensearch Replicas.
+
+Alternatively, the `values.yaml` and `values-production.yaml` file can be 
+directly updated if the Helm chart was pulled locally. For example,
+
+```console
+helm pull oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/timesketch --untar
+```
+
+Then make changes to the downloaded `values.yaml` and once done, install the 
+chart with the updated values.
+
+```console
+helm install my-release ../timesketch
+```
+
+## Persistence
+
+The Timesketch deployment stores data at the `/mnt/timesketchvolume` path of the
+container and stores configuration files at the `/etc/timesketch` path of the container. 
+
+Persistent Volume Claims are used to keep the data across deployments. This is 
+known to work in GCE and minikube. See the Parameters section to configure the 
+PVC or to disable persistence.
+
+## Upgrading
+
+If you need to upgrade an existing release to update a value, such as persistent 
+volume size or upgrading to a new release, you can run [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/). 
+For example, to set a new release and upgrade storage capacity, run:
```

Comment:
Similar comment about documenting resizing vs. deleting and recreating here (as above with Turbinia).